app/code/community/Ebizmarts/MailChimp/etc/system.xml

```diff
@@ -112,7 +113,8 @@
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
                             <can_be_empty>1</can_be_empty>
-                            <comment><![CDATA[Synchronize Magento's General Subscription Audience with this Mailchimp audience.]]></comment>
+                            <comment>
```

Comment:
This blank lines after the start tag cause failure.

Reply:
Solved.